doc/source/reference/style.rst

```diff
@@ -24,6 +24,7 @@ Styler properties
 
    Styler.env
    Styler.template
+   Styler.template_latex
```

Comment:
can we rename template -> template_html

Reply:
there is a section in the user_guide on subclassing Styler and hooking into the jinja2 block names to extend it. If you change the name of the template it might break that feature for existing users.

Reply:
OT: i dont' care really about breaking down stream subclassing. if possible you can deprecate, but if it makes sense to normalize naming then let's do it (usually these things are easy to do in a backwards compatible way in any event). We also likey don't have any/many tests on subclassing, so maybe hard to tell.